tests/main/proxy/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Ensure that the core.proxy.* settings are honored
+
+systems: [ubuntu-18.04-64]
+
+restore: |
+    snap set core proxy.https=
+    apt autoremove --purge -y squid
+
+prepare: |
+    rm -rf /var/log/squid/*
```

Comment:
Do you need to restart squid after purging the logs?

Reply:
This is just to ensure we have a clean state in case a machine is reused.